tests/test_render_markdown.py

```diff
@@ -137,3 +137,53 @@ def test_explicit_column(metadata):
         database="mydatabase",
         datasette=Datasette([], metadata=metadata),
     )
+
+
+def test_extensions():
```

Comment:
This test uses the https://python-markdown.github.io/extensions/tables/ extension.